.gitignore

```diff
@@ -2,5 +2,4 @@
 **/*.rs.bk
 *.idea/
 *.iml
-Cargo.lock
```

Comment:
While it can run as a binary, Rebalancer uses sharkspotter as a library, so we added it to the .gitignore.

Maybe we could use workspaces to separate the two.

Reply:
Ok, I didn't realize it was used also as a library. That makes sense now. It looks like rebalancer manager is using an older tag of sharkspotter and there would be some required changes to rebalancer to update to a later version so maybe I'll defer this work until I've found some ways to improve performance. I'll undo this for now.

Reply:
My changes for MANTA-5274 will be updating the sharkspotter version to 0.13.0.